burner-ui/src/Pages/ConfirmPage/ConfirmPage.tsx

```diff
@@ -5,29 +5,24 @@ import Button from '../../components/Button';
 import Page from '../../components/Page';
 import LineItem from '../../components/LineItem';
 
-const ConfirmPage: React.FC<BurnerContext & RouteComponentProps> = ({ history, assets, actions, pluginData }) => {
+const ConfirmPage: React.FC<BurnerContext & RouteComponentProps> = ({ history, assets, actions }) => {
```

Comment:
Looks like this PR is rolling back some updates, can this file be brought up to date?

Reply:
Brought back up to date